pages/Auto-Enchanter.md

```diff
@@ -0,0 +1,10 @@
+The Auto Enchanter is an [Electric Machine](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) which can be used to add Enchanted Books to tools and weapons.
+
+Together with the [Auto Disenchanter](https://github.com/Slimefun/Slimefun4/wiki/Auto-Disenchanter) you can use this to disenchant [Reinforced Armor](https://github.com/Slimefun/Slimefun4/wiki/Reinforced-Armor) to obtain Protection IX and Unbreaking IX books and put those on diamond or netherite gear.
```

Comment:
I think this could be reworded? Sounds like it's the only purpose when it's just an example/suggestion, don't know if it's just me reading it that way. I would mirror the explanation of the auto disenchanter and then add to both a line like the above as an example usage or tip or whatever